default.json

```diff
@@ -6,10 +6,10 @@
     ":rebaseStalePrs",
     ":disableDependencyDashboard",
     ":automergeBranch",
-    ":automergePatch",
     ":automergeMinor",
     "github>cucumber/renovate-config:gemspec",
     "github>cucumber/renovate-config:disable-perl"
   ],
-  "prHourlyLimit": 0
-}
\ No newline at end of file
+  "prHourlyLimit": 0,
+  "separateMinorPatch": true
```

Comment:
We can probably remove this.

I don't immediately see a use for it.

Reply:
Awesome, will remove - should reduce some PRs across the organisation